opg-php-fpm-1604/docker/php/xdebug-enable.ini

```diff
@@ -0,0 +1,11 @@
+zend_extension=/usr/lib/php/20151012/xdebug.so
+xdebug.default_enable=1
+xdebug.max_nesting_level=200
+xdebug.profiler_enable=0
+xdebug.remote_autostart=1
+xdebug.remote_connect_back=0
+xdebug.remote_enable=1
+xdebug.remote_handler=dbgp
+xdebug.remote_mode=req
+xdebug.remote_log=/var/log/xdebug_remote.log
```

Comment:
to get the logs into kibana, a beaver config needs to be added for the log file.